docs/support.md

```diff
@@ -3,17 +3,66 @@ hide:
 - navigation
 ---
 
-# Support
+# General Support
 
-The best way to ask general questions about a particular use cases is to email us at [support+ci@cirruslabs.org](mailto:support+ci@cirruslabs.org).
+The best way to ask general questions about particular use cases is to email our support team at [support+ci@cirruslabs.org](mailto:support+ci@cirruslabs.org).
+Our support team is trying our best to response ASAP but there is no guarantee on a response time unless your organization enrolls in [Priority Support](#priority-support).
 
 If you have a feature request or noticed lack of some documentation please feel free to [create a GitHub issue](https://github.com/cirruslabs/cirrus-ci-docs/issues/new/choose).
-Our support team will answer it by replying or updating documentation.
+Our support team will answer it by replying to the issue or by updating documentation.
 
-## Migration Assistance
+# Priority Support
 
-Cirrus Labs can help your team with migration to Cirrus CI. Our team will analyze your current needs and workflow in order
-to not only perform the migration, but also perform optimizations along the way to make sure your team's workflow is as 
-optimal as it can possibly be.
+In addition to the general support we provide a *Priority Support* option with guaranteed response times and some extra
+care like bi-yearly checkins to make sure Cirrus CI roadmap is aligned with your company's needs.
 
-Our team has experience of optimizing developer experiences at companies like Airbnb and Twitter. Contact us at [support+migration@cirruslabs.org](mailto:support+migration@cirruslabs.org).
+| Severity | Support Impact                                                                               | First Response Time SLA | Hours | How to Submit                     |
+|----------|----------------------------------------------------------------------------------------------|-------------------------|-------|-----------------------------------|
+| 1        | Emergency (Completely unusable)                                                              | 30 minutes              | 24x7  | Please trigger an urgent issue    |
+| 2        | Highly Degraded (Important features unavailable or extremely slow; No acceptable workaround) | 4 hours                 | 24x5  | Please use priority email address |
+| 3        | Medium Impact                                                                                | 8 hours                 | 24x5  | Please use priority email address |
+| 4        | Low Impact                                                                                   | 24 hours                | 24x5  | Please use priority email address |
+
+`24x5` means period of time from 9AM on Monday till 5PM on Friday in EST timezone.
+
+<!-- markdownlint-disable MD037 -->
+??? note "Support Impact Definitions"
+    * **Severity 4** - Questions or Clarifications around features or documentation (24 hours) Minimal or no Business Impact. 
```

Comment:
Do you think it'd be easier to cross-reference these definition with the table if they'd be in ascending order too?